### PR TITLE
Fix ProductID and ReportID of New 1060 Plus

### PR DESCRIPTION
**Pull request by X9VoiD**

### CODE REVIEW

**Comment by InfinityGhost** [comment]
Lets make sure that other users of the New 1060 Plus don't have this same situation.

File: OpenTabletDriver/Configurations/Huion/New 1060 Plus.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "Name": "Huion New 1060 Plus",
   "DigitizerIdentifier": {
     "VendorID": 9580,
-    "ProductID": 109,
+    "ProductID": 110,
```
Comment (InfinityGhost) [inline OpenTabletDriver/Configurations/Huion/New 1060 Plus.json:5]: How about we make this change for the alternate digitizer?

**Review by InfinityGhost — CHANGES_REQUESTED** [review]

**Comment by InfinityGhost** [comment]
#350 might be related to this

**Comment by X9VoiD** [comment]
The diagnostics from #350 confirmed that the product id is correct, but the input report length is different.
We could make an alternate identifier, one for InputReportLength 8 and one for InputReportLength 12

**Comment by X9VoiD** [comment]
Oh... uhh. We can't make an alternate identifier for MaxPressure lol. We should really rework the TabletProperties now

**Comment by InfinityGhost** [comment]
Update this to conform to latest tablet configuration changes

**Comment by X9VoiD** [comment]
PR updated to new config format

**Comment by InfinityGhost** [comment]
#350, what should we do about it?

**Comment by X9VoiD** [comment]
Converted the PR to draft since we can't test the configurations on Windows (device stream seems to not open), and not verified yet.

**Comment by X9VoiD** [comment]
Superseded by #562

**Pull request closed without merging**